remotewrite/sender/backoff_test.go

```diff
@@ -0,0 +1,300 @@
+// Copyright The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/prometheus/compliance/remotewrite/sender/targets"
+)
+
+// TimestampTrackingReceiver wraps MockReceiver to track request timestamps.
+type TimestampTrackingReceiver struct {
+	*MockReceiver
+	mu         sync.Mutex
+	timestamps []time.Time
+}
+
+// NewTimestampTrackingReceiver creates a receiver that tracks request timestamps.
+func NewTimestampTrackingReceiver(baseReceiver *MockReceiver) *TimestampTrackingReceiver {
+	return &TimestampTrackingReceiver{
+		MockReceiver: baseReceiver,
+		timestamps:   make([]time.Time, 0),
+	}
+}
+
+// RecordTimestamp records the current time for a request.
+func (ttr *TimestampTrackingReceiver) RecordTimestamp() {
+	ttr.mu.Lock()
+	defer ttr.mu.Unlock()
+	ttr.timestamps = append(ttr.timestamps, time.Now())
+}
+
+// GetTimestamps returns all recorded timestamps.
+func (ttr *TimestampTrackingReceiver) GetTimestamps() []time.Time {
+	ttr.mu.Lock()
+	defer ttr.mu.Unlock()
+	return append([]time.Time{}, ttr.timestamps...)
+}
+
+// TestBackoffRequired validates that senders implement a backoff mechanism when retrying.
```

Comment:
We already have description, let's remove this comment? And on all tests with desc?

We can move it to 
```suggestion
// TestBackoffRequired is a compliance test. See t.Attr("description").
```